sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/aggregate/AggregateExpressionSuite.scala

```diff
@@ -31,4 +32,23 @@ class AggregateExpressionSuite extends SparkFunSuite {
     assert(expected == actual, s"Expected: $expected. Actual: $actual")
   }
 
+  test("test regr_count input types") {
```

Comment:
This is too trivial and the type check framework should have already been well tested. I think we can remove it.

Reply:
OK